fpage/templates/fpage/index.html

```diff
@@ -20,11 +20,36 @@ <h2>{{ album.album_title }}</h2>
           <button class="btn btn-outline fas fa-heart" data-bookmark-url="{% url 'fpage:fav-album' %}" data-album-id="{{ album.id }}" onclick='favouriteAlbum(this)' role="button"></button>
           {% endif %}
           <!-- Delete-->
-          <a href="{%url 'fpage:album-delete' album.id  %}" class="btn btn-info" role="button">Delete</a>
+          {% comment %} <a href="{% url 'fpage:album-delete' album.id %}" class="btn btn-info" role="button">Delete</a> {% endcomment %}
```

Comment:
remove this comment

Reply:
when i click on delete nothing is happening that was a problem. please have a look at the conversation between me and shivam in #53 it will tell what problem i am facing.

Reply:
I am unable to solve the issue. Shivam will look into it after 2-3 days and let you know.

Reply:
ok sure